src/components/Supporters/index.jsx

```diff
@@ -0,0 +1,227 @@
+import React from "react";
+import styles from "./styles.module.css";

```

Comment:
I don't know if this is a rule in the JavaScript world, but most of us will be looking at it with a Python mindset. So I wonder if the following is a good idea.

Python imports are usually sorted alphabetically, and grouped in three blocks:

- Standard library
- 3rd party modules
- Project modules

If we applied that style here we would have:

- React, Docusaurus
- Theme?
- Our css styles

Would that make sense?